po/pt.po

```diff
@@ -342,7 +342,7 @@ msgstr "Monitor:"
 
 #: ../src/Grids/PlankGrid.vala:116
 msgid "The monitor that Plank is displayed on"
-msgstr ""
+msgstr "O monitor em que a barra é exibida"
```

Comment:
Same here.

Reply:
Yes I know there are several times the word 'Barra' instead of 'Plank' but those those are to stay that way on purpose.
I am also the Portuguese language translator of the Plank program in the launchpad and the elementary OS in Weblate and it makes more sense for Portuguese users to be like this.

[Anotação 2020-01-19 115617](https://user-images.githubusercontent.com/16929048/72680539-d0ce6b80-3ab2-11ea-8410-4aca899f46eb.png)